locales/en/moves.json

```diff
@@ -5,13 +5,16 @@
   "steps": {
     "person_search": {
       "heading": "Who is being moved?",
+      "heading_add_to_allocation": "Who do you want to add to this allocation?",
```

Comment:
The language and naming of these things feels a little disjointed. We're now referring to "add_to_allocation" where previously we've used the term "assign" and related it to the move, not the allocation.

Might be worth keeping a consistent context of `assign` when talking about how we add a person to a move. That's what is really happening underneath, regardless of how we suggest to a user that it is part of the allocation.